docs/examples/drupal7/index.md

```diff
@@ -259,4 +257,4 @@ tugboat-build: build
 #tugboat-init: packages createdb drupalconfig importdb stagefileproxy build cleanup
 #tugboat-update: importdb stagefileproxy build cleanup
 #tugboat-build: build
-```
\ No newline at end of file
+</code></pre>
```

Comment:
This should be moved to the end of the previous line to avoid extra whitespace at the end of the rendered code block.

Reply:
Can you show me what you mean? I'm not seeing this on the Tugboat preview.

Reply:
Nope, you're right. I was mis-remembering what I did in my proof-of-concept.